circle.yml

```diff
@@ -56,11 +56,14 @@ test:
             parallel: true
         - ./bin/console doctrine:schema:drop --force --env=test_mysql:
             parallel: true
-        - ./bin/console doctrine:schema:create --env=test_mysql:
+        - ./bin/console doctrine:migrations:migrate -n --env=test_mysql:
```

Comment:
Could you also update the Makefile to use migrations in test (task `tfp`) instead of schema:create ?

Reply:
Done.